### PR TITLE
Draft of first day of class

### PR DESCRIPTION
**Pull request by jsjrobotics**

### CODE REVIEW

File: lessons/env-and-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+- title: Environment Setup
+- tags: setup
+
+#Standards:
+
+#Objectives
+Install Intelli J
```
Comment (alexhsamuel) [inline lessons/env-and-tools/README.md]: Put dashes at the front of each line to turn this into a bullet list.

File: lessons/env-and-tools/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+[Android Source Code](https://source.android.com/)
+[Github](https://www.github.com)
+[Git] (https://git-scm.com/docs/user-manual.html)
+
+# Lecture
+
+Welcome to Access Code 3.3
+
+## IDE setup
+Intall Intelli J
```
Comment (alexhsamuel) [inline lessons/env-and-tools/README.md]: Bullet list.

**Comment by rina-lam-goldfield** [comment]
Note that on launch day there will also be lots of other "welcome to access code" speeches. Better to dive directly into coding.

**Comment by alexhsamuel** [comment]
LGTM.

**Comment by jsjrobotics** [comment]
Will add the Slack intro as agreed with @abassawo @ramonaharrison  and will then be ready to merge

**Comment by jsjrobotics** [comment]
@abassawo  if you approve please merge

**Pull request merged**